doc/ws.md

```diff
@@ -0,0 +1,121 @@
+# Websockets interface
+
+## What is this?
+
+This interface is a new way to talk with Orion using websockets, this is point to improve the comunication with a faster and efficient protocol, now we have a persistent connection with a open socket. 
```

Comment:
I'm not able to locate this particular change in the diff tab... could you add a link to help to locate it, please?